### PR TITLE
fix error link summary

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: en/summary.rst [hunk 1]
```diff
@@ -97,7 +95,8 @@ Hosting guide
 .. _`Create a block`: /en/developer_guide/block_creation.rst
 .. _`Device detection`: /en/developer_guide/multi_device.rst
 .. _`Twig extensions`: /en/developer_guide/twig_extensions.rst
-.. _`Provisionning`: /en/hosting_guite/server_provisionning.rst
+.. _`Provisionning`: /en/hosting_guide/server_provisionning.rst
+.. _`Logs`: /en/hosting_guide/logs.rst
```
Comment (nicolasThal) [inline en/summary.rst]: could you move this line to respect the tree ?

Comment (alavieille) [inline en/summary.rst]: what tree ? i don't understand where you want to move this line

**Comment by alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**